### PR TITLE
Migrating to Seaborn

### PR DESCRIPTION
**Pull request by m0hit-kumar**
<h1> Migrating to Seaborn</h1>
Contributor:
<p><b >Name: </b><a  href="https://github.com/m0hit-kumar">Mohit Kumar </a> </p>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,7 +20,8 @@
     ],
     python_requires='>=3.6',
     install_requires=[
-        'matplotlib'
+        'matplotlib',
+        "seaborn"
```
Comment (hot9cups) [inline setup.py]: Could probably make both of them be in single quotes(or double quotes)

**Review by hot9cups — COMMENTED** [review]
This looks fantastic! I'm quite happy, well done!

The code is great, and so is the code cleanup.
I've still got to run some tests with the changes made in requirements.txt, I had some bugged out installation but I think that might be a local issue so I'll have to check.

Also, since we're migrating towards using seaborn, I don't think we'd still need to list matplotlib as a required library?

File: setup.py (hunk 1 above)
Comment (hot9cups) [reply setup.py]: Also, shouldn't searborn alone suffice? I don't see any parts that were matplotlib that weren't translated.

Comment (m0hit-kumar) [reply setup.py]: plt.show() is used that is of matplotlib lib

**Comment by m0hit-kumar** [comment]
> This looks fantastic! I'm quite happy, well done!
> 
> The code is great, and so is the code cleanup. I've still got to run some tests with the changes made in requirements.txt, I had some bugged out installation but I think that might be a local issue so I'll have to check.
> 
> Also, since we're migrating towards using seaborn, I don't think we'd still need to list matplotlib as a required library?

we are using plt.show() from matplotlib ,we dont have show() in seaborn

**Pull request closed without merging**